neo4j/neo4j-bundle/0.3/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "bundles": {
+        "Neo4j\\Neo4jBundle\\": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "NEO4J_DEFAULT_CONNECTION_SCHEME": "bolt",
```

Comment:
Do we need `_DEFAULT_` to be part of env var names? We usually don't do that in other recipes.

Reply:
Hello @javiereguiluz, thanks for your feedback. If the `_DEFAULT_CONNECTION_` prefix is not something common, I think that providing a such information would be helpful, don't you think?

Reply:
I think it is needed. The bundle supports multiple connections.